_pkgdown.yml

```diff
@@ -0,0 +1,49 @@
+url: https://lintr.r-lib.org
+template:
+  bootstrap: 5
+
+  includes:
+    in_header: |
+      <script defer data-domain="lintr.r-lib.org,all.tidyverse.org" src="https://plausible.io/js/plausible.js"></script>
+
+reference:
+- title: Linting
+  contents:
+  - lint_dir
+  - lint_file
+  - lint_package
+  - linters_with_defaults
+  - linters_with_tags
+  - read_settings
+  - exclude
+  - modify_defaults
+  - checkstyle_output
+
+- title: Individual linters
+  contents:
+  - all_undesirable_functions
+  - ends_with("linter")
+
+- title: Linter bunldes
+  contents:
+  - ends_with("linters")
+
+
+- title: Pacakge development
```

Comment:
Maybe rename to "Linter development" or "Custom linters"?

Reply:
Done, let me know if I didn't correctly interpret your suggestions as to what other functions should live in here.